concent_api/common/tests/test_decorators.py

```diff
@@ -278,3 +281,20 @@ def task():
 
         mock_format_exc.assert_called_once()
         mock_error.assert_called_once()
+
+
+@override_settings(
+    DETECT_NESTED_TRANSACTIONS=True,
+)
+class NonNestingAtomicDecoratorTestCase(TransactionTestCase):
+
+    def test_that_non_nesting_atomic_decorator_will_raise_exception_if_transaction_is_nested_using_same_database(self):
+        with non_nesting_atomic(using='control'):
+            with self.assertRaises(ConcentPendingTransactionError):
+                with non_nesting_atomic(using='control'):
+                    pass
+
+    def test_that_non_nesting_atomic_decorator_will_not_raise_exception_if_transaction_is_nested_using_different_databases(self):
```

Comment:
```
************* Module common.tests.test_decorators
R:297, 4: Method could be a function (no-self-use)
```
I'm guessing that's because there is no `assert` in this test. Can we check at least that there are no `Exception` being raised?

Reply:
> Can we check at least that there are no Exception being raised?

Unfortunately not, so I guess the only way is to silence it.

Reply:
Okey, I'm fine with that.